usdt-macro/src/lib.rs

```diff
@@ -79,10 +79,11 @@ pub fn dtrace_provider(item: proc_macro::TokenStream) -> proc_macro::TokenStream
         _ => panic!("DTrace provider must be a single literal string filename"),
     };
     let source = if filename.ends_with(".d") {
-        fs::read_to_string(&filename).expect(&format!(
+        let dir = std::env::var("CARGO_MANIFEST_DIR").unwrap();
+        let path = Path::new(&dir).join(&filename);
```

Comment:
If that env variable isn't available, would it make sense to attempt to use the bare filename as the path?  I'm not sure if anyone would attempt to use this by building with `rustc(1)` directly.

Reply:
Doesn't hurt to make this `unwrap_or_else` or similar.